conjure-core/src/main/java/com/palantir/conjure/defs/validator/EndpointDefinitionValidator.java

```diff
@@ -457,6 +458,19 @@ public void validate(EndpointDefinition definition) {
         }
     }
 
+    @com.google.errorprone.annotations.Immutable
+    private static final class NoUnsupportedHttpMethodValidator implements ConjureValidator<EndpointDefinition> {
+        @Override
+        public void validate(EndpointDefinition definition) {
+            Preconditions.checkState(
+                    HttpMethod.values().contains(definition.getHttpMethod()),
```

Comment:
I think we can simplify this into `!HttpMethod.Value.UNKNOWN.equals(definition.getHttpMethod().value())`. Our generated enums wrap standard java enums, where we use an additional `UNKNOWN` value to represent any string value that isn't declared in the enum.

Reply:
Got it, thanks for the suggestion 😄.